### PR TITLE
Add example of postgres database cluster and instance with scheduled PITR backup

### PR DESCRIPTION
**Pull request by morozovalekseywot**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ description: |-
 - Fix panic on VKCS Kubernetes API error in vkcs_kubernetes_clustertemplates data source
 - Fix an error when changing several Kubernetes cluster node groups in parallel
 - Fix an error when reading the state of a compute instance with the build status
+- Fix unexpected state error when the database cluster was performing backups after creation
+- Add example of postgres database cluster and instance with scheduled PITR backup
```
Comment (ftersin) [inline CHANGELOG.md:18]: do not mention doc changes

File: examples/db/cluster/with_scheduled_pitr_backup/main.tf [hunk 2]
```diff
@@ -0,0 +1,32 @@
+resource "vkcs_db_cluster" "pg_with_backup" {
+  name = "pg-with-backup"
```
Comment (ftersin) [inline examples/db/cluster/with_scheduled_pitr_backup/main.tf:2]: we use tf-example suffix in object names to let a user know the origin of the object (if he forgets to call tf destroy)

File: examples/db/instance/with_scheduled_pitr_backup/main.tf [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  size        = 8
+  volume_type = "ceph-ssd"
+  disk_autoexpand {
+    autoexpand    = true
+    max_disk_size = 1000
+  }
+
+  cloud_monitoring_enabled = true
```
Comment (ftersin) [inline examples/db/instance/with_scheduled_pitr_backup/main.tf:22]: lets remove this

File: examples/db/instance/with_scheduled_pitr_backup/base.tf [hunk 4]
```diff
@@ -0,0 +1,29 @@
+data "vkcs_compute_flavor" "db" {
```
Comment (ftersin) [inline examples/db/instance/with_scheduled_pitr_backup/base.tf:1]: could we use links to base manifests instead?

File: CHANGELOG.md (hunk 1 above)
Comment (morozovalekseywot) [reply CHANGELOG.md]: done

File: examples/db/instance/with_scheduled_pitr_backup/main.tf (hunk 3 above)
Comment (morozovalekseywot) [reply examples/db/instance/with_scheduled_pitr_backup/main.tf]: done

File: examples/db/cluster/with_scheduled_pitr_backup/main.tf [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+  volume_size = 10
+  volume_type = "ceph-ssd"
+
+  network {
+    uuid = vkcs_networking_network.db.id
+  }
+
+  backup_schedule {
+    name           = "three_hours_backup"
```
Comment (ftersin) [inline examples/db/cluster/with_scheduled_pitr_backup/main.tf:22]: the same tf-example, pls

Comment (morozovalekseywot) [reply examples/db/cluster/with_scheduled_pitr_backup/main.tf]: done

**Comment by paaanic** [comment]
@morozovalekseywot Rebase, please.

**Pull request merged**